### DISCUSSION TITLE
I dont understand this error

### DISCUSSION THREAD

**Discussion by KamiCS9** [games647/FastLogin#1172] [Q&A] [answered]
Im having this error while trying to set up fastlogin on the bungee server
![image](https://github.com/games647/FastLogin/assets/161782537/98f94a99-38ce-4efb-8887-376094df1436)

**Comment by TuxCoding** [comment]
Please post the complete error.

**Reply by KamiCS9** [reply]
I have a different problem now, but theres the complete error

**Reply by TuxCoding** [reply, marked as answer]
The error means you have an invalid configuration. It indicates that it tries to read a number (like a port number), but it got interpreted as text. This can happen if you put it into quotes or mixed it with textual characters.

**Reply by KamiCS9** [reply]
Thanks, i hadnt had the time to check again, it was something on the config.yml i already changed it and now it works
